### PR TITLE
Feature/tslint generated code

### PR DESCRIPTION
**Pull request by Vyperus**

### CODE REVIEW

File: generators/app/templates/module/module.ts [hunk 1]
```diff
@@ -1,10 +1,8 @@
 import {CommonModule} from '@angular/common';
 import {NgModule} from '@angular/core';
-
+import {<%=moduleName%>MockRepository} from './repositories/<%=moduleFileName%>.mock.repository';
+import {<%=moduleName%>DefaultService} from './services/<%=moduleFileName%>.default.service';
```
Comment (S3bastianGriesa) [inline generators/app/templates/module/module.ts]: Order should be
```
DefaultService
MockRepository
```

File: generators/app/templates/module/repository.mock.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+import {Http} from '@angular/http';
+import {environment} from 'environment';
+import {Observable} from 'rxjs/Observable';
+import {I<%=moduleName%>Repository} from './<%=moduleFileName%>.repository.interface';
+
+
+@Injectable()
+export class <%=moduleName%>MockRepository implements I<%=moduleName%>Repository {
+
+  constructor(private http: Http) {}
```
Comment (S3bastianGriesa) [inline generators/app/templates/module/repository.mock.ts]: No parameter properties

Comment (S3bastianGriesa) [inline generators/app/templates/module/repository.mock.ts]: And no http in mockrepository

File: generators/app/templates/module/module.ts (hunk 1 above)
Comment (Vyperus) [inline generators/app/templates/module/module.ts]: No

**Pull request merged**